bin/create_release.py

```diff
@@ -0,0 +1,290 @@
+#!.venv/bin/python
+
+import argparse
+import bugsy
+import os
+import subprocess
+import sys
+from git import Repo
+
+DEFAULTS = {
+    "dev": {
+        "bugzilla": "bugzilla-dev.allizom.org",
+        "bugzillaRest": "https://bugzilla-dev.allizom.org/rest",
+        "product": "Loop",
+    },
+    "prod": {
+        "bugzilla": "bugzilla.mozilla.org",
+        "bugzillaRest": "https://bugzilla.mozilla.org/rest",
+        "product": "Hello (Loop)"
+    }
+}
+
+oldApiKey = None
+
+
+def runProcess(cmd, cwd, errorMessage):
+    print "runProcess ", cmd
+    p = subprocess.Popen(cmd, cwd=cwd)
+
+    result = p.wait()
+
+    if result:
+        print >> sys.stderr, errorMessage % result
+        sys.exit(1)
+
+
+def checkTag(repo, version):
+    realTag = "v" + version
+
+    try:
+        result = repo.tags[realTag]
+    except:
+        print >> sys.stderr, "Could not find tag %s for version %s" % (realTag, version)
+        return None
+
+    print "Tag %s found..." % realTag
+    return result
+
+
+def loginToBugzilla(useSystem, username, apikey):
+    bugzillaUrl = DEFAULTS[useSystem]["bugzillaRest"]
+
+    print "Logging into bugzilla %s..." % bugzillaUrl
+
+    bugzilla = bugsy.Bugsy(username=username, api_key=apikey, bugzilla_url=bugzillaUrl)
+
+    print "Done..."
+
+    return bugzilla
+
+
+def getInput(message):
+    print message
+    result = raw_input("(y/n): ").rstrip("\r\n")
+
+    while result != "y" and result != "n":
+        print "Please answer 'y' or 'n'!"
+        result = raw_input("(y/n): ").rstrip("\r\n")
+
+    return result
+
+
+def releaseStandalone(bugzilla, repo, useSystem, version, username):
+    print "Creating a bug for standalone"
+
+    print "Please enter the previous release version:"
+    previousRelease = raw_input().rstrip("\r\n")
+
+    print "Please enter required timescales, e.g.:"
```

Comment:
Calling it a timescale seems a bit confusing.  Maybe just "Please enter any requests for the deployers, e.g.:"?